src/python/WMCore/WorkerThreads/BaseWorkerThread.py

```diff
@@ -142,6 +142,15 @@ def __call__(self, parameters):
             # to get the right name
             myThread = threading.currentThread()
 
+            if hasattr(self.component.config, "General") and \
+               hasattr(self.component.config.General, "central_logdb_url") and \
+               hasattr(self.component.config, "Agent"):
+                from WMCore.Services.LogDB.LogDB import LogDB
```

Comment:
I still have to digest the usage of this logDB :) but my first comment would be about the location of this import. Can't we import it from the top of this module?